geomesa-spark/geomesa-spark-jts/pom.xml

```diff
@@ -31,15 +31,13 @@
         </dependency>
         <dependency>
             <groupId>com.vividsolutions</groupId>
-            <artifactId>jts-core</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.geotools</groupId>
-            <artifactId>gt-data</artifactId>
+            <artifactId>jts-io</artifactId>
```

Comment:
we don't currently pull in jts-io - looks like this will need a cq

Reply:
Sure! I also plan on using that module for the Row to GeoJSON ticket